src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,16 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given thread. The following restrictions apply:
```

Comment:
I'm wondering whether we should call the below list "restrictions" as they're more "possibilities"? 🤔 E.g. for the threads, there's rather no restrictions as a thread can be pretty much only either active or inactive :D, and another: tags are case sensitive is not rather a restriction but a fact, you know what I mean :D

Reply:
The word 'restrictions' I based on the fact that that is how the intro sentences to the different bullet lists were introduced. They did not have the same info. However, some were similar https://developers.livechat.com/docs/messaging/agent-chat-api/rtm-reference#chat-users
However, I get your point, so maybe - ...Attention should be paid to the following information."?

Reply:
What about "rules"?

Reply:
With 'rules', I think I have similar worries like Oliwia above with 'restrictions' - '...Active and inactive chats can be tagged..' or '...The tags’ list can be found in the LiveChat  Application.' - it more sounds like a fact/info. 
So maybe 
"...Attention should be paid to the following rules and information:..."?
or
"The following rules and information apply..." (If we want to follow the rest of the bullet lists' intros)

I cannot come up with one word describing all the points...

Reply:
"terms and conditions apply" 😂 :trollface: 

Can we drop this "preface" altogether and just do a new "paragraph" with all this additional info?

Reply:
I would stick to the bullet list as it will be easier for the reader to go through all this info. Also, each point is different, so I believe a bullet list would be more effective.

Reply:
still, some points are related and can be grouped as separate "paragraphs"

Reply:
Plus we can replace "restrictions" with "info" if needed.

Reply:
Yes, I was thinking to change 'restrictions' for 'infrmation'

Reply:
It looks like the description will shrink after all the corrections :) Let's see if it still makes sense to keep the bullet list. If there are at least 3 bullet points, then I think it's worth it. If fewer, then it's better to format it as a paragraph.

Reply:
Both methods will have less than 3 bullet points, so it makes no sense to keep the bullet list.  Paragraphs will be better in that case.